### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by ryuzakyl**
This is a first approach to working with Pull Requests

### CODE REVIEW

File: programming-solutions/Program.cs [hunk 1]
```diff
@@ -15,10 +15,14 @@ static void Main(string[] args)
             //DoubleName();
             //TotalTimeDoubleName();
             //TotalTimeDoubleName1();
+
+            // take least time that took the user to write his name
```
Comment (madelinkind) [inline programming-solutions/Program.cs]: Looks good

File: programming-solutions/Program.cs [hunk 2]
```diff
@@ -15,10 +15,14 @@ static void Main(string[] args)
...
             shorterTime();
         }
 
-
+        /// <summary>
+        /// Handles user's name and age
+        /// </summary>
```
Comment (madelinkind) [inline programming-solutions/Program.cs]: Add return type

File: programming-solutions/Program.cs (hunk 1 above)
Comment (ryuzakyl) [reply programming-solutions/Program.cs]: Cool!

File: programming-solutions/Program.cs (hunk 2 above)
Comment (ryuzakyl) [reply programming-solutions/Program.cs]: Done

**Pull request merged**